.github/workflows/tag_image_push_rhel.yml

```
@@ -96,8 +96,12 @@ jobs:
       RELEASE_VERSION: ${{ needs.prepare.outputs.RELEASE_VERSION }}
 
     runs-on: ubuntu-latest
+
+    # Workaround _suspected_ race conditions on the RedHat server-side publish
+    concurrency: rhel
```

`concurrency: rhel` will be removed, yes?

---

No - if we retagged a bunch of images, we still don't want to be running multiple publishes at once.

---

but this will mean that if we add another jdk (a third one) we will silently skip the rebuild of the one jdk version